src/nix_auto_follow/cli.py

```diff
@@ -64,6 +64,11 @@ def get_url(self) -> str:
                 return f"github:{original['owner']}/{original['repo']}{rev_or_ref}"
             case "gitlab":
                 return f"gitlab:{original['owner']}/{original['repo']}{rev_or_ref}"
+            case "sourcehut":
+                return f"sourcehut:{original['owner']}/{original['repo']}{rev_or_ref}"
+            case "tarball":
+                # This is hella complicated
+                return ""
```

Comment:
Why did you add it ?
Should we remove it now and let it fail.

Reply:
The comment could be removed but it does let cases such as the below pass:
`    "flake-schemas": {
      "locked": {
        "lastModified": 1721999734,
        "narHash": "sha256-G5CxYeJVm4lcEtaO87LKzOsVnWeTcHGKbKxNamNWgOw=",
        "rev": "0a5c42297d870156d9c57d8f99e476b738dcd982",
        "revCount": 75,
        "type": "tarball",
        "url": "https://api.flakehub.com/f/pinned/DeterminateSystems/flake-schemas/0.1.5/0190ef2f-61e0-794b-ba14-e82f225e55e6/source.tar.gz"
      },
      "original": {
        "type": "tarball",
        "url": "https://flakehub.com/f/DeterminateSystems/flake-schemas/%3D0.1.5.tar.gz"
      }
    },`

Although we may need to remove it. What do you think?

Reply:
I can go add a test case for the open pull request for the tarball. That should resolve this tarball issues for DeterminateSystems.

Reply:
thanks